src/pages/Sign/sections/SignUp.tsx

```diff
@@ -3,9 +3,13 @@ import {
   TranslationsConsumer,
   TranslationsPlaceholderConsumer,
 } from '@hedviginsurance/textkeyfy'
+import { SessionContainer } from 'containers/SessionContainer'
 import { Field, Form, Formik } from 'formik'
+import gql from 'graphql-tag'
 import * as React from 'react'
+import { Mutation, Subscription } from 'react-apollo'
 import styled from 'react-emotion'
+import { Redirect } from 'react-router'
```

Comment:
just fyi, this should be `react-router-dom` :) `react-router` works but only for legacy compatibility reasons and its deprecated